code/game/objects/items/cards_ids.dm

```diff
@@ -275,13 +275,45 @@ update_label("John Doe", "Clowny")
 	item_state = "orange-id"
 	lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
-	assignment = "Prisoner"
-	registered_name = "Scum"
+	assignment = "Convict"
 	var/goal = 0 //How far from freedom?
 	var/points = 0

```

Comment:
Might as well remove `goal` and `points` if they're never used.

That said, *are they* never used? I think we do have lavaland mining as punishment going on somewhere; would be nice if this didn't break that functionality.

Reply:
These are still referenced in the gulag. Unconfigured prisoner IDs act the same as before so it shouldn't be affected. We should look at the gulag at some point.